results/cmd/api/main.go

```diff
@@ -81,3 +96,23 @@ func (s *server) CreateTaskRun(ctx context.Context, req *pb.CreateTaskRunRequest
 	}
 	return taskrunFromClient, nil
 }
+
+func (s *server) DeleteTaskRun(ctx context.Context, req *pb.DeleteTaskRunRequest) (*empty.Empty, error) {
+	database := s.db
+	statement, err := database.Prepare("DELETE FROM taskrun WHERE uid = ?")
+	if err != nil {
+		log.Fatalf("failed to create delete statement: %v", err)
+	}
+	results, err := statement.Exec(req.GetUid())
+	if err != nil {
+		log.Fatalf("failed to execute delete statement: %v", err)
+	}
+	affect, err := results.RowsAffected()
```

Comment:
Same as Get, I'm not sure if the `uid` field is what we want to use here. LogID, or even (namespace, task run name) may make more sense.

Reply:
I am thinking about it as well. We can discuss about it next week